internal/pkg/cli/env_init.go

```diff
@@ -90,12 +89,19 @@ func newInitEnvOpts(vars initEnvVars) (*initEnvOpts, error) {
 		initEnvVars:   vars,
 		projectGetter: store,
 		envCreator:    store,
-		envDeployer:   cloudformation.New(profileSess),
 		projDeployer:  cloudformation.New(defaultSession),
 		identity:      identity.New(defaultSession),
-		envIdentity:   identity.New(profileSess),
 		profileConfig: cfg,
 		prog:          termprogress.NewSpinner(),
+		initProfileClients: func(o *initEnvOpts) error {
+			profileSess, err := session.NewProvider().FromProfile(o.EnvProfile)
+			if err != nil {
+				return fmt.Errorf("Cannot create session from profile %s: %w", o.EnvProfile, err)
```

Comment:
```suggestion
				return fmt.Errorf("create session from profile %s: %w", o.EnvProfile, err)
```
The recommended guideline for error strings from Go:  
> Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Another one that we follow is avoiding words like "failed" or "cannot" in error strings as usually they're redundant in the context of errors :) 

https://github.com/uber-go/guide/blob/master/style.md#error-wrapping

Reply:
Gotcha, thanks for the links!